setup.py

```diff
@@ -3,16 +3,16 @@
 
 setup(
     name="OpenOversight",
-    version="1.0",
-    description="Oversight of Police Departments",
+    version="0.6.7",
+    description="OpenOversight is a Lucy Parsons Labs project to improve law "
+    + "enforcement accountability through public and crowdsourced data.",
     author="redshiftzero",
     author_email="jen@redshiftzero.com",
     classifiers=(
         "Natural Language :: English",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3 :: Only",
     ),
-    python_requires=">=3.5",
+    python_requires=">=3.11",
     install_requires=["flask", "werkzeug", "Flask-WTF", "psycopg2", "sqlalchemy"],
```

Comment:
These are VERY much out of sync, but I'm a little unsure of how to update them other than just reading from the `requirements.txt` file.

Reply:
Since it's so out of date and we only use the `requirements.txt` file, I'm tempted to delete the `install_requires` line: https://github.com/lucyparsons/OpenOversight/blob/9bef4baedb08365df53c603340a7547c711b5b1d/DEPLOY.md?plain=1#L5-L9